changelogs/fragments/1055_add_vrf_global_module.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
it is a major change but the tag apt here would be minor changes i believe since its not affecting the end users or breaking any changes there thats why the tag minor

Reply:
This is a minor change keeping in mind the impact on existing playbooks.